Gulpfile.js

```diff
@@ -162,6 +168,9 @@ gulp.task('watch', function() {
 // Set Develop task
 gulp.task('develop', defaults.develop_tasks);
 
+// Set develop-vagrant task
+gulp.task('develop-vagrant', defaults.develop_tasks_vagrant);
```

Comment:
@patrickfweston is `develop-vagrant` and `defaults.develop_tasks_vagrant` running the same thing?

Reply:
Being honest, I've forgotten how the pieces fit together since I made this PR. It appears to me that I'm missing the definition of the default variable `defaults.develop_tasks_vagrant` in this PR. I believe it should be set up as such:

`develop-vagrant` is an ease-of-use task that calls the tasks defined in `defaults.develop_tasks_vagrant`. This is setup in the same way that `develop` calls the tasks defined in `defaults.develop_tasks`:

```
defaults.develop_tasks = ['sass', 'sculpin', 'watch'];
```


The only difference is that we use `spress-serve-vagrant` to serve the prototype so it runs with the appropriate config.